front-end/src/molecules/CollaboratorItem.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { TrashCanIcon } from "../atoms/icons";
+import "./CollaboratorItem.css";
+
+const CollaboratorItem = ({ text, remove, pending }) => {
```

Comment:
One suggestion is to just make the className conditional instead of haveing the style code in the function.